ofrak_core/Dockerstub

```diff
@@ -61,7 +61,7 @@ RUN cd /tmp && \
 
 # Install Jefferson
 WORKDIR /tmp
-RUN wget https://bootstrap.pypa.io/pip/get-pip.py && python3.9 get-pip.py && python3.7 get-pip.py && rm get-pip.py
+RUN wget https://bootstrap.pypa.io/pip/get-pip.py && python3.9 get-pip.py && python3.8 get-pip.py && rm get-pip.py
 RUN python3.9 -m pip install jefferson
```

Comment:
The upgrade to this Python version juxtaposed with the next line makes me think maybe we should just bump to 3.9 while we are at it, and remove the weird alternative python version install. It also gives more of a buffer for the next Python version EoL. What do you think?

Reply:
Hm, might be a reasonable thing to do - let me test and see if anything else breaks from it.

Reply:
@EdwardLarson unfortunately, with Python 3.9, some of the `test_ofrak/service/serialization_service/test_pjson.py` tests fail in `ofrak_core`. Let's maybe do this one for now, and worry about 3.9 later?